jobs/JRADAR_LEVEL2

```diff
@@ -130,7 +104,12 @@ postmsg "$msg"
 
 set -x
 #export NUMPROC=$LSB_DJOB_NUMPROC
+# JY check here
+echo "DEBUG: NCPUS=$NCPUS"
+
+# JY export NUMPROC=32
```

Comment:
echo statements for debugging don't need to be present in production

Reply:
done